src/pages/home.js

```diff
@@ -229,6 +229,8 @@ export default function Home() {
       } catch (error) {
         // TODO: implement global alert or notification to handle the error
         console.error(error);
+      } finally {
+        importFile.current.value = null;
```

Comment:
Could we reset the value of input through an event instead of using a ref? The "importAction" should not be tied to any specific input element.

Reply:
You're right, using event is better. Thanks! https://github.com/coseeing/Access8MathWeb/pull/31/commits/5ed3c2af2b5e516c399d59a255edf8ddf0376a63